### PR TITLE
ETS-1310: QS: Tab, Input field & button styling

### PR DESCRIPTION
**Pull request by brunogeronimo**
- Creates `tab-menu` and `dropdown` components into the styleguide;
- Includes new examples for the new components into Patternlab;
- Adds `.vscode` folder and `.DS_Store` into `.gitignore` file;

### CODE REVIEW

**Review by dknx01 — APPROVED** [review]

File: source/_patterns/02-organisms/06-menu/01-menu.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Menu
```
Comment (FTI-Touristik-GmbH) [inline source/_patterns/02-organisms/06-menu/01-menu.md]: Menu is the wrong name. These are Tabs

File: source/scss/components/_dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.dropdown {
+  padding: 0;
+
+  .dropdown-menu {
+    border: 0;
+    border-radius: 0;
+    box-shadow: 0 4px 5px 2px transparentize($black, .9);
```
Comment (virgilchiriac) [inline source/scss/components/_dropdown.scss:7]: should have a shadow mixin

Comment (brunogeronimo) [reply source/scss/components/_dropdown.scss:7]: This shadow was abstracted using variables for now. Issue #45 was created to organize shadows in a better way in the future.

File: source/_patterns/02-organisms/06-tabs/01-tabs.mustache [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                Inactive tab with link
+            </a>
+        </li>
+    </ul>
+</div>
+<div class="tab-content">
+    <div class="tab-pane show active">
+        {{ plainText }}
+    </div>
+</div>
```
Comment (virgilchiriac) [inline source/_patterns/02-organisms/06-tabs/01-tabs.mustache]: should include molecule instead of tab-content...

**Review by virgilchiriac — APPROVED** [review]

**Review by cllagosr — APPROVED** [review]

**Pull request merged**